plugwise/helper.py

```diff
@@ -122,14 +122,14 @@ def schemas_schedule_temp(schedules, name) -> float | None:
                 return schema_list[i][2]
 
 
-def types_finder(data):
+def types_finder(data) -> set:
     """Detect types within locations from logs."""
-    types = set()
+    types: set = set()
```

Comment:
Addin typing here doesn't add anything, it is already assigned a set, so it can be inferred.

```suggestion
    types = set()
```